<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Message</class>
 <widget class="QMainWindow" name="Message">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonNO">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>390</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Закрыть</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="messageBG">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>552</width>
      <height>461</height>
     </size>
    </property>
    <property name="text">
     <string>bg</string>
    </property>
   </widget>
   <widget class="QLabel" name="messageText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>messageText</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>messageBG</zorder>
   <zorder>pushButtonNO</zorder>
   <zorder>messageText</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
